src/test/java/teammates/test/cases/ui/browsertests/InstructorHomePageUiTest.java

```diff
@@ -296,8 +296,62 @@ public void testRemindActions(){
 
     }
 
-    public void testPublishUnpublishActions(){
-        //TODO add test for publishing and unpublishing feedback sessions
+    public void testPublishUnpublishActions() {
+        ______TS("publish action: AWAITING feedback session");
+        
+        homePage.verifyUnclickable(homePage.getPublishLink(feedbackSession_AWAITING.courseId, feedbackSession_AWAITING.feedbackSessionName));
+
+        ______TS("publish action: OPEN feedback session");
+        
+        homePage.clickFeedbackSessionPublishLink(feedbackSession_OPEN.courseId, feedbackSession_OPEN.feedbackSessionName);
+        ThreadHelper.waitFor(1000);
```

Comment:
This is probably a bad idea... 

Try to find something on the page to wait for.